packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -188,15 +188,15 @@ export async function getBookings({
   if (filters?.afterStartDate) {
     bookingWhereInputFilters.afterStartDate = {
       startTime: {
-        gte: dayjs(filters.afterStartDate).utc().endOf("day").toDate(),
+        gte: dayjs(filters.afterStartDate).utc().toDate(),
```

Comment:
This needs to be 
```
gte: dayjs.utc(filters.afterStartDate).toDate(),
```

because if in query params we have `afterStartDate=2025-01-15` and we log what i suggest and what you have:
```
console.log("asap 1", dayjs.utc(filters.afterStartDate).toDate());
console.log("asap 2", dayjs(filters.afterStartDate).utc().toDate());
```

then in the logs we see:
```
@calcom/web:dev: asap 1 2025-01-15T00:00:00.000Z
@calcom/web:dev: asap 2 2025-01-14T23:00:00.000Z
```

aka that with current implementation it is `2025-01-14T23:00:00.000Z`